compose/compose-file/index.md

```diff
@@ -2061,6 +2062,9 @@ Compose implementations MUST set `com.docker.compose.project` and `com.docker.co
 If set to `true`, `external` specifies that this network’s lifecycle is maintained outside of that of the application.
 Compose Implementations SHOULD NOT attempt to create these networks, and raises an error if one doesn't exist.
 
+If `external` is set to `true` and network configuration has other but `name` attributes set, considering resource is
+not managed by compose lifecycle, Compose Implementations SHOULD reject a Compose file as invalid.
```

Comment:
I'm not sure about this passage:
```suggestion
If `external` is set to `true` and network configuration has other attributes set besides `name` , resource consideration is not managed by the Compose lifecycle. In these cases, Compose Implementations SHOULD reject the Compose file as invalid.
```

Reply:
I believe what it's saying is that for anything `external: true`, the only other allowable attribute is `name`. If other attributes are defined, Compose Implementations SHOULD reject the Compose file as invalid.

Perhaps:
```
If `external` is set to `true`, the resource is not managed by Compose. If `external` is set to `true` and the network configuration has other attributes set besides `name` , Compose Implementations SHOULD reject the Compose file as invalid.
```